barchart/cue.mod/module.cue

```diff
@@ -2,3 +2,12 @@ module: "github.com/perses/plugins/barchart@v0"
 language: {
 	version: "v0.11.0"
 }
+source: {
+	kind: "git"
+}
+deps: {
+	"github.com/perses/perses/cue@v0": {
+		v:       "v0.0.1-test"
```

Comment:
version to be changed when it'll be time to merge on main.
We'll have to align versioning of the module with the versioning of the binary. The same should apply to the versioning of the plugin schemas too.